extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -1138,6 +1138,151 @@ describe
           yield onBindReadyAndSetState(env, bind, {foo: 'bar'});
           expect(foo.textContent).to.not.equal('bar');
         });
+
+        describe('AmpEvents.FORM_VALUE_CHANGE', () => {
+          it('should dispatch FORM_VALUE_CHANGE on [disabled] changes', () => {
+            const element = createElement(
+              env,
+              container,
+              '[disabled]="foo"',
+              'input'
+            );
+            const spy = sandbox.spy(element, 'dispatchEvent');
+
+            return onBindReadyAndSetState(env, bind, {foo: 'disabled'}).then(
+              () => {
+                expect(spy).to.have.been.calledOnce;
+                expect(spy).calledWithMatch({
+                  type: AmpEvents.FORM_VALUE_CHANGE,
+                  bubbles: true,
```

Comment:
I feel this whole request: 
{
                  type: AmpEvents.FORM_VALUE_CHANGE,
                  bubbles: true,
                }
can be extracted to be a constant, so you don't need to have it across all tests.